prime.py

```diff
@@ -1,33 +1,72 @@
 import timeit
+import unittest

```

Comment:
not sure why unittest is needed throughout the code, using the time benchmark should be sufficient